config/environment.js

```diff
@@ -81,14 +81,15 @@ module.exports = function(environment) {
     // RESTAdapter Settings
     ENV.APP.SOCKETIO_WEBSERVICE_URL = 'http://localhost:1337/goodcity';
     ENV.APP.API_HOST_URL = 'http://localhost:3000';
+    ENV.APP.BASE_HOST_URL = 'http://localhost:4203';
```

Comment:
Can we change this name to `STOCK_APP_HOST_URL` for readability and clarity purposes?

Reply:
Yes, that is more clear.